pyanaconda/modules/storage/bootloader/bootloader.py

```diff
@@ -447,6 +450,16 @@ def collect_requirements(self):
 
         return requirements
 
+    @property
+    def use_bootupd(self):
+        """Whether bootupd is enabled"""
+        return self._use_bootupd
+
+    def set_use_bootupd(self):
+        """Install the bootloader using https://github.com/coreos/bootupd"""
+        self._use_bootupd = True
+        self.set_bootloader_mode(BootloaderMode.SKIPPED)
+
```

Comment:
This is the "implementation", in the bootloader "module", but it needs to be added to the "interface", too. In the file `bootlader_interface.py`, add the same thing, CamelCase-named, touching  `self.implementation.<name-from-module>`. Also, it's a D-Bus property, so in the interface should be a property, not a function. Check `bootloader_mode` and `BootloaderMode` to see an example how this is done.

(The interface members are translated to D-Bus automagically)